pyart/io/output_to_geotiff.py

```diff
@@ -0,0 +1,318 @@
+"""
+pyart.io.write_grid_geotiff
+====================
+
+Write a Py-ART Grid object to a GeoTIFF file.
+
+.. autosummary::
+    :toctree: generated/
+
+    write_grid_geotiff
+    _get_rgb_values
+    _create_sld
+
+"""
+
+from __future__ import print_function, division
+import numpy as np
+import matplotlib.pyplot as plt
+import matplotlib.colors as colors
+import os
+import shutil
+from ..exceptions import MissingOptionalDependency
+try:
+    from osgeo import gdal
+    IMPORT_FLAG = True
+except ImportError:
+    IMPORT_FLAG = False
+
+
+def write_grid_geotiff(grid, filename, field, rgb=False, level=None,
+                       cmap='viridis', vmin=0, vmax=75, color_levels=None,
+                       warp=False, sld=False):
+    """
+    Write a Py-ART Grid object to a GeoTIFF file.
+
+    The GeoTIFF can be the standard Azimuthal Equidistant projection used
+    in Py-ART, or a lat/lon projection on a WGS84 sphere. The latter is
+    typically more usable in web mapping applications. The GeoTIFF can
+    contain a single float-point raster band, or three RGB byte raster bands.
+    The former will require an SLD file for colorful display using standard
+    GIS or web mapping software, while the latter will show colors
+    "out-of-the-box" but lack actual data values. The function also can
+    output an SLD file based on the user-specified inputs. User can specify
+    the 2D vertical level to be output. If this is not specified, a 2D
+    composite is created. User also can specify the field to output.
+
+    This function requires GDAL Python libraries to be installed. These are
+    available via conda; e.g., 'conda install gdal'
+
+    Parameters
+    ----------
+    grid : pyart.core.Grid object
+        Grid object to write to file.
+    filename : str
+        Filename template for the GeoTIFF. The field name will be appended
+        prior to the suffix. For example, if the filename template given is
+        'radar.tif', and the field name is 'reflectivity', then
+        the final file will be called 'radar_reflectivity.tif'.
+    field : str
+        Field name to output to file. This also will be used to create the
+        final GeoTIFF filename.
+
+    Other Parameters
+    ----------------
+    rbg : bool, optional
+        True - Output 3-band RGB GeoTIFF
```

Comment:
Sphinx doesn't like something about these True/False blocks and is giving a warning.  I think a blank line is needed after each option, one after the True block, one after the False, but I'm not sure.  Easiest method I know to check if these work is to try building the docs.  I can do this before merging if you would prefer.